arm-machinelearning/2016-05-01-preview/swagger/commitmentPlans.json

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
After some offline discussion, we are going to leave them as separate for now and consider a composite client for a future update.